pkg/controller/installation/marketplace/manager.go

```diff
@@ -104,8 +104,6 @@ func (m *MarketplaceManager) CreateSubscription(ctx context.Context, serverClien
 			return err
 		} else if err == nil {
 			logrus.Infof("catalog source config created")
```

Comment:
Can we remove this change. There is a subtle bug here which I am fixing with a follow up pr, but this should return here as the catalog source config may not be ready yet

Reply:
I ran into this on my tests.
Can we check that the catalogsource is ready by checking the resource?

Just returning doesn't guarantee its ready either

Reply:
Yeah with the current design, we might need to return an error here so that the reconcile stops and the next time through we can check again. I am ok with this merging as is for now actually as I am looking at improving it in my next pr